### ISSUE TITLE
3pMedia library accessibility: focus order is inconsistent

### ISSUE THREAD

**Issue by miina** [swissspidy/web-stories-wp#10642]
## Feature Description
When tabbing from a focused media item, canvas gets the focus. However, when doing Shift-Tab then, the 3p media provider link gets the focus instead.

This behaviour should be consistent -- either focus on the attribution after tabbing in focused media item or skip focusing on it.

## How to reproduce

1. Open the 3p media library
2. Tab into a media item
3. Tab again
4. See the focus on Canvas
5. Now click "Shift + Tab"
6. The focus is on the Unsplash attribution
